module/x/gravity/types/types.go

```diff
@@ -67,23 +66,26 @@ func GetPrefixFromBech32(bech32str string) (string, error) {
 
 // GetNativePrefixedAccAddressString treats the input as an AccAddress and re-prefixes the string
 // with this chain's configured Bech32AccountAddrPrefix
```

Comment:
Changed this function to use bech32ibc instead of the static config, since the rest of our logic will be working with bech32ibc's NativeHrp value